openfast_io/openfast_io/FAST_writer.py

```diff
@@ -786,8 +786,8 @@ def write_AeroDyn15(self):
             self.write_AeroDyn15Coord()
         
         if self.fst_vt['AeroDyn15']['Wake_Mod'] == 3:
-            if self.fst_vt['AeroDyn15']['UAMod'] == 2:
-                raise Exception('OLAF is called with unsteady airfoil aerodynamics, but OLAF currently only supports UAMod == 1') #TODO: need to check if this holds true now
+            if self.fst_vt['AeroDyn15']['UA_Mod'] == 2:
+                raise Exception('OLAF is called with unsteady airfoil aerodynamics, but OLAF currently only supports UA_Mod == 1') #TODO: need to check if this holds true now

```

Comment:
I haven't read much of this code, but I don't understand this comment.  The UnsteadyAero module doesn't allow `UA_Mod` to be 1, including when it is called from OLAF.

Reply:
Thank for catching that! I think I missed that with new AD15 API updates.

Reply:
AeroDyn*